ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -9,16 +9,41 @@
         data-test-isReindexing />
     </div>
   {{/if}}
-  <div
-    class="selectable-card-container {{if isSecondary 'secondary' 'primary'}}"
-    data-test-selectable-card-container={{if isSecondary 'secondary' 'primary'}}>
-    {{yield (hash
-      card=(component componentToRender replicationDetails=replicationDetails)
-    )}}
-    {{#if (not isSecondary)}}
-      {{yield (hash secondaryCard=(component 'known-secondaries-card'))}}
-    {{/if}}
-  </div>
+  {{#if isSummaryDashboard}}
+    <div class="summary-state">
+      <h6 class="title is-5 {{if (get (cluster-states summaryState) "isOk") "" "has-text-danger"}}" data-test-error>state</h6>
+      {{#unless (get (cluster-states summaryState) "isOk")}}
+        <AlertInline
+          @type="danger"
+          @message="The cluster is {{summaryState}}. Please check your server logs." />
+      {{else}}
+        <p class="has-text-grey is-size-8">The current operating state of both Disaster Recovery and Performance. This value should be ‘running.’ If the value is idle, it indicates an issue and needs to be investigated.</p>
+      {{/unless}}
+      <h2 class="title is-3" data-test-summary-state>{{summaryState}}
+        {{#if (get (cluster-states summaryState) "isOk")}}
+          <Icon @glyph={{get (cluster-states summaryState) "glyph"}} class="{{if (get (cluster-states summaryState) "isOk") "has-text-success" "has-text-danger"}}" data-test-icon />
+        {{/if}}
+      </h2>
+    </div>
+    <div
+      class="selectable-card-container summary secondary"
```

Comment:
does this need the `secondary` class? 'cause it might not necessarily be a secondary, right?

Reply:
oh yeah, i think changing the rule to apply to `&.secondary, &.summary { //grid stuff }` would be a great fix. that way the classes of the element accurately describe what it is.